requirements.txt

```diff
@@ -0,0 +1,5 @@
+PyYAML >= 5.1
```

Comment:
Is it common to have spaces in there? I haven't been using requirements.txt too much, but those I saw before don't have spaces in there, and `pip freeze` also doesn't give spaces in there.

Reply:
It works for me, but I can of course change it in case some edge cases occur